Sources/PointFree/Database.swift

```diff
@@ -301,7 +303,7 @@ private func fetchSubscription(id: Database.Subscription.Id) -> EitherIO<Error,
     ORDER BY "created_at" DESC
     LIMIT 1
     """,
-    [id.rawValue.rawValue]
+    [id.rawValue.uuidString]
```

Comment:
I removed the `RawRepresentable` conformance I think I added to UUID. Seems like a bad idea now.

Reply:
haha yeah